.eslintrc

```diff
@@ -27,13 +27,15 @@
     "valid-jsdoc": 2
   },
   "globals": {
+    "jest": false,
```

Comment:
Globals can be easily set by `env`: https://www.npmjs.com/package/eslint-plugin-jest